Support for running against recent JDKs

Currently it appears that a bloop server running on a recent JDK (9/10) can successfully build a program against this JVM,

But when attempting to run the compiled output with `bloop run`, the following error arises:

```[E] Boxed Error
[T] java.util.concurrent.ExecutionException: Boxed Error
[T]     scala.concurrent.impl.Promise$.resolver(Promise.scala:83)
[T]     scala.concurrent.impl.Promise$.scala$concurrent$impl$Promise$$resolveTry(Promise.scala:75)
[T]     scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:280)
[T]     scala.concurrent.Promise.tryFailure(Promise.scala:108)
[T]     scala.concurrent.Promise.tryFailure$(Promise.scala:108)
[T]     scala.concurrent.impl.Promise$DefaultPromise.tryFailure(Promise.scala:183)
[T]     monix.eval.internal.TaskRunLoop$$anon$1.onError(TaskRunLoop.scala:281)
[T]     monix.eval.internal.TaskRunLoop$.monix$eval$internal$TaskRunLoop$$loop$1(TaskRunLoop.scala:242)
[T]     monix.eval.internal.TaskRunLoop$RestartCallback$1.onSuccess(TaskRunLoop.scala:119)
[T]     monix.eval.Task$.$anonfun$forkedUnit$2(Task.scala:1463)
[T]     java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1135)
[T]     java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[T]     java.base/java.lang.Thread.run(Thread.java:844)
[T] Caused by: java.lang.UnsatisfiedLinkError: Error looking up function 'Java_java_lang_UNIXProcess_forkAndExec': /home/rom1dep/Binaires/jdk/jdk-10.0.1/lib/libjava.so: undefined symbol: Java_java_lang_UNIXProcess_forkAndExec
[T]     com.sun.jna.Function.<init>(Function.java:245)
[T]     com.sun.jna.NativeLibrary.getFunction(NativeLibrary.java:566)
[T]     com.sun.jna.NativeLibrary.getFunction(NativeLibrary.java:542)
[T]     com.sun.jna.NativeLibrary.getFunction(NativeLibrary.java:528)
[T]     com.sun.jna.Native.register(Native.java:1777)
[T]     com.sun.jna.Native.register(Native.java:1370)
[T]     com.zaxxer.nuprocess.internal.LibJava8.<clinit>(LibJava8.java:36)
[T]     com.zaxxer.nuprocess.linux.LinuxProcess.start(LinuxProcess.java:109)
[T]     com.zaxxer.nuprocess.linux.LinProcessFactory.createProcess(LinProcessFactory.java:40)
[T]     com.zaxxer.nuprocess.NuProcessBuilder.start(NuProcessBuilder.java:266)
[T]     bloop.exec.Forker$.$anonfun$run$2(Forker.scala:166)
[T]     monix.eval.internal.TaskRunLoop$.monix$eval$internal$TaskRunLoop$$loop$1(TaskRunLoop.scala:187)
[T]     monix.eval.internal.TaskRunLoop$RestartCallback$1.onSuccess(TaskRunLoop.scala:119)
[T]     monix.eval.Task$.$anonfun$forkedUnit$2(Task.scala:1463)
[T]     java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1135)
[T]     java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[T]     java.base/java.lang.Thread.run(Thread.java:844)```

--- a/project/Dependencies.scala
+++ b/project/Dependencies.scala
@@ -21,7 +21,7 @@ object Dependencies {
   val ipcsocketVersion = "1.0.0"
   val monixVersion = "2.3.3"
   val circeVersion = "0.9.3"
-  val nuprocessVersion = "1.2.3"
+  val nuprocessVersion = "1.2.4"
   val shapelessVersion = "2.3.3-lower-priority-coproduct"
   val scalaNativeVersion = "0.3.7"
   val scalaJs06Version = "0.6.23"

